services/database/relational/tests/ccc-rdms-c2.feature

```diff
@@ -5,20 +5,41 @@ Feature: Secure Database Access Control
 This feature verifies various security measures in the database management system, including disabling default credentials, ensuring only authorized roles have access, and denying access attempts using default credentials.
 """
 
-@CCC.RDMS.C2.TR02.T01
+@CCC.RDMS.C2.TR01.T01
 Scenario: Ensure that only authorized roles can access database resources
    Given a user with an authorized role
    When the user tries to access the database resources
    Then the user should be granted access to the database resources
 
-@CCC.RDMS.C2.TR02.T02
+@CCC.RDMS.C2.TR01.T02
 Scenario: Ensure that unauthorized roles cannot access database resources
    Given a user with an unauthorized role
    When the user tries to access the database resources
    Then the user should be denied access to the database resources
 
-@CCC.RDMS.C2.TR02.TR03
+@CCC.RDMS.C2.TR01.TR03
 Scenario: Confirm that access attempts using default credentials are denied
    Given the database management system has default credentials
    When an access attempt is made using default credentials
    Then the access attempt should be denied
+
+
+@CCC.RDMS.C2.TR02
+Feature: Secure Database Access Control with Local users
+
+"""
+This feature targets database configurations where a local user is defined and granted permissions to interact with the database system.  
+"""
+
+@CCC.RDMS.C2.TR02.TR01
+Scenario: Ensure that only authorized local accounts exist in the database and are restricted to accessing the data they need
+   Given a local database with user accounts that may be used for application access
+   When auditing local accounts 
+   Then only expected local accounts exist in the database
+   And each account is properly scoped to the expected permissions
+
+@CCC.RDMS.C2.TR02.TR02
+Scenario: Ensure that authorized accounts only have the minimum neccessary permissions to perform their task
+   Given that local accounts must be granted certain permissions to perform certain operations on the database system
```

Comment:
```suggestion
   Given a local database with user accounts
```

Since this is all still pre-release and open to further iteration, I'm not personally concerned with getting it perfect prior to merge... but adding some notes anyhow.

We should keep in mind the `Given` step is intended to represent a base state or precondition that can be checked prior to the tests being run.

In this example, I'm imagining that we want to use this step to report whether there are non-admin accounts present _before_ going on to report whether all non-admin accounts are correctly provisioned.